ludwig/schema/utils.py

```diff
@@ -687,13 +738,20 @@ def _jsonschema_type_mapping(self):
                 else []
             )
 
-            return {"oneOf": oneof_list, "title": self.name, "description": description, "default": default}
+            return {
```

Comment:
Not part of the PR, but would you mind adding this line above after `numeric_option` is declared?

```
if not is_integer: numeric_option["format"] = "float"
```

Reply:
Done.